apps/events-helsinki/src/pages/_error.tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouter } from 'next/router';
+import React from 'react';
+
+/**
+ * Due to translation problem: https://github.com/i18next/next-i18next/issues/1020
+ * let's redirect always to the pages/error.tsx page.
+ */
+const NextErrorPage = () => {
+  const router = useRouter();
+  React.useEffect(() => {
+    if (router.pathname !== '/error') {
```

Comment:
do we still need this check?

Reply:
no, it should be removed, because if we get there, and do not navigate elsewhere, it would lead to an empty white page (null).